staging/src/k8s.io/apiextensions-apiserver/test/integration/subresources_test.go

```diff
@@ -424,21 +389,16 @@ func TestValidationSchemaWithStatus(t *testing.T) {
 	}
 
 	// fields other than properties in root schema are not allowed
-	noxuDefinition := newNoxuValidationCRDs(apiextensionsv1beta1.NamespaceScoped)[0]
-	noxuDefinition.Spec.Subresources = &apiextensionsv1beta1.CustomResourceSubresources{
-		Status: &apiextensionsv1beta1.CustomResourceSubresourceStatus{},
-	}
-	_, err = fixtures.CreateNewCustomResourceDefinition(noxuDefinition, apiExtensionClient, dynamicClient)
-	if err == nil {
-		t.Fatalf(`unexpected non-error, expected: must not have "additionalProperties" at the root of the schema if the status subresource is enabled`)
-	}
```

Comment:
why is this removed?

Reply:
> why is this removed?

a side effect of trying to make the schema eventually creatable.  Some tests relied on having a single schema for all versions.  Others wanted two versions.

Reply:
It amazes me that we have an invalid schema in a standard fixture.

Reply:
ok, found it: for structural schemas we have this:

```
	if g.AdditionalProperties != nil {
		if lvl == rootLevel {
			allErrs = append(allErrs, field.Forbidden(fldPath.Child("additionalProperties"), "must not be used at the root"))
		}
```

In other words, we forbid `additionalProperties` even for CRDs without status subresource. So removing this extra test is fine. Please also remove the now invalid comment above the removal.